assets/laion.yaml

```diff
@@ -1,5 +1,59 @@
 ---
 
+- type: dataset
+  name: LAION-400M
+  # General
+  organization: LAION
+  description: LAION-400M is a dataset with CLIP-filtered 400 million image-text
+    pairs, their CLIP embeddings and kNN indices that allow efficient similarity
+    search. This dataset is entirely openly, freely accessible.
+  created_date:
+    value: 2021-08-20
+    explanation: The date the [[blog post]](https://laion.ai/blog/laion-400-open-dataset/)
+      was released.
+  url: https://laion.ai/blog/laion-400-open-dataset/
+  datasheet: https://laion.ai/blog/laion-400-open-dataset/
+  modality: text (English) and image
+  size: 400M image-text pairs
+  sample: []
+  analysis: ''
+  # Construction
+  dependencies: [CommonCrawl]
```

Comment:
should there be a dependency on CLIP due to the filtering?

Reply:
You're right, will add CLIP as a dependency